src/test/java/gov/healthit/chpl/aqa/stepDefinitions/ChplAPISteps.java

```diff
@@ -24,4 +34,50 @@ public void verifyContentUnderCertifiedHealthITProductListingAndURL(final String
         assertEquals(actualText, text);
         assertEquals(actualURL, urlLink);
     }
+
+    /**
+     * Assert that title should be CHPL API.
+     * @param title expected as CHPL API
+     */
+    @Then("^CHPL API page title should be \"([^\"]*)\"$")
+    public void chplApiPageTitleShouldBe(final String title) {
+        String chplAPITitle = getDriver().getTitle();
+        Assert.assertEquals(chplAPITitle, title);
+    }
+
+    /**
+     * Click all the controller elements link.
+     * @param controllerName as the name of Controller
+     */
+    @When("^I click on \"([^\"]*)\" link$")
+    public void iClickOnLink(final String controllerName) {
+        WebElement link = ChplAPIPage.controllerLink(getDriver(), controllerName);
+        ((JavascriptExecutor) getDriver()).executeScript("arguments[0].click();", link);
+    }
+
+    /**
+     * Assert that APIs from webpage is present in the example API Endpoints.
+     * @param apiEndpoints as the details of controller
+     */
+    @Then("^list of \"(.*)\" operations should be displayed without deprecated calls$")
+    public void listOfOperationsShouldBeDisplayed(final String apiEndpoints) {
+        WebDriver driver = getDriver();
+        // Get element from the webpage into the list
+        List<WebElement> listWebElements = ChplAPIPage.controllerElementList(driver);
+        // Creating a list to hold all the API listed on the webpage by swagger
+        List<String> apiListFromPage = new ArrayList<>();
+        //Splitting string into tokens of APIs separated by "--" and putting into a list
+        List<String> apiListFromEndpoints = Arrays.asList(apiEndpoints.split("--"));
+        // Checking if the API from the webpage is present in the API Endpoints
+        for (WebElement wElement : listWebElements) {
+        // Get text of the Web Elements list
+            String apiName = wElement.getText();
+            apiListFromPage.add(apiName);
+            assertTrue(apiListFromEndpoints.contains(apiName), "API is deprecated/newly added api not found in the dataset:-" + apiName);
```

Comment:
Don't need either of the two previous lines, and this one could just be
```suggestion
            assertTrue(apiListFromEndpoints.contains(apiName), "Unknown API found: " + apiName);
```